.github/workflows/CI.yml

```diff
@@ -33,7 +33,6 @@ jobs:
           restore-keys: ${{ runner.os }}-test-artifacts
       - uses: julia-actions/julia-buildpkg@v1
       - uses: julia-actions/julia-runtest@v1
-        continue-on-error: ${{ matrix.version == 'nightly' }}
```

Comment:
Did this actually work?

Reply:
It just shows the step and job as a pass. GHA still lacks a proper allowed failure mechanism...